### ISSUE TITLE
Jellyfin error , version 0.3.1

### ISSUE THREAD

**Issue by sparklyballs** [jellyfin/jellyfin-kodi#95]
**Describe the bug**

When 0.3.1 is installed , it will not load, get error message advising to check log...

**To Reproduce**

Installing version 0.3.1 on a fresh install of Kodi/Libreelec

**Expected behavior**

for it to work ?
**Logs**

```
2019-09-28 21:28:41.570 T:2656031600   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
- NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
Error Type: <type 'exceptions.SyntaxError'>
Error Contents: EOL while scanning string literal (full_sync.py, line 544)
Traceback (most recent call last):
File "/storage/.kodi/addons/plugin.video.jellyfin/service.py", line 24, in <module>
from entrypoint import Service
File "/storage/.kodi/addons/plugin.video.jellyfin/resources/lib/entrypoint/__init__.py", line 23, in <module>
from service import Service
File "/storage/.kodi/addons/plugin.video.jellyfin/resources/lib/entrypoint/service.py", line 17, in <module>
import library
File "/storage/.kodi/addons/plugin.video.jellyfin/resources/lib/library.py", line 16, in <module>
from full_sync import FullSync
File "/storage/.kodi/addons/plugin.video.jellyfin/resources/lib/full_sync.py", line 544
LOG.debug('Mystery function item type: {}'.format('items[0][1]))
^
SyntaxError: EOL while scanning string literal
-->End of Python script error report<--
```
**Screenshots**

**System (please complete the following information):**
 - Kodi Version: [e.g. 18.4]
 - Via LibreELEC-RPi4 v9.1.501

**Additional context**

**Comment by sparklyballs** [comment]
Update, temporarily downgrading to 0.3.0 and then allowing the system to auto-update the plugin , seems to resolve the error.

**Comment by oddstr13** [comment]
Fixed with #93

**Comment by oddstr13** [comment]
Should be fixed with the release of v0.3.2
